projects/SCION/programs/basic/tests/basic.nix

```
@@ -0,0 +1,28 @@
+{
+  sources,
+  ...
+}:
+
+{
+  name = "scion";
+
+  nodes = {
+    machine =
+      { ... }:
+      {
+        imports = [
+          sources.modules.ngipkgs
+          sources.modules.services.scion
```

Use the program example instead of the service's since this is testing the programs

```suggestion
          sources.modules.programs.scion
```